helm/templates/notification-service/application-config.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.notificationService.enabled}}
+{{- if and (not .Values.hypertraceAlertEngine.enabled) .Values.notificationService.enabled }}
```

Comment:
why do we need two conditions? Can't we just work with one?

Reply:
Can we just use one `.Values.notificationService.enabled`?

Reply:
a case may arise when both `.Values.notificationService.enabled` and `.Values. hypertraceAlertEngine.enabled` are true, I wanted to avoid that

Reply:
But, that means, that how someone wants to deploy, right?

Reply:
Ah! I see that notification service is also part of hypertrace-alert-engine

Reply:
I'm still not following - I discussed this in more detail in https://github.com/Traceableai/harness/pull/220#discussion_r745068205 , but my expectation would be to have one block of config for the aggregated service form, and a separate block of config for the separate deployment form. Otherwise, it's really unclear which values are picked up and which aren't 

i.e. (bad naming, but shows the idea)

```yaml
  deploymentType: aggregated # or individual

  aggregated:
    # any config that's only relevant when running aggregate deployment would fall under here
    resources: ...

   individual:
    # any config that's only relevant when doing distinct deployments would fall under here
     notificationService:
       enabled: true
       resources: ...
     otherService:
       resources: ...

   # could put other config, relevant to either deployment format back at top level e.g. app config
```

Reply:
Used the same naming, let me know. 
Also had to use independent bool var for kafka-topic chart (due to limitation of what can be specified in the dependency condition)